sentry-android/src/main/java/io/sentry/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -138,6 +145,29 @@ public void helpBuildingEvent(EventBuilder eventBuilder) {
         return contexts;
     }
 
+    private static String[] getProGuardUuids(Context ctx) {
+        try {
+            AssetManager assets = ctx.getAssets();
```

Comment:
Agreed, we should just do the same as these: https://github.com/getsentry/sentry-java/blob/132d0ee0f2b795cbcc5d25ffeb8e21958b90d709/sentry-android/src/main/java/io/sentry/android/event/helper/AndroidEventBuilderHelper.java#L43